subsystems/intakesubsystem.py

```diff
@@ -0,0 +1,143 @@
+from enum import Enum, auto
+from commands2 import Subsystem
+from wpilib import RobotBase
+from wpimath.geometry import Rotation2d
+from ntcore import NetworkTableInstance
+from util.simtalon import Talon
+from util.simcoder import CTREEncoder
+from util.angleoptimize import intakeAccountForSillyEncoder
+
+import constants
+
+
+class IntakeSubsystem(Subsystem):
+    class IntakeState(Enum):
+        Intaking = auto()
+        Score = auto()
```

Comment:
score vs scoring isn't that easily clear, but doesn't affect my review of this code